package network;

import org.junit.Test;

import java.io.IOException;
import java.nio.channels.SocketChannel;

import static org.junit.Assert.assertTrue;

/**
 * Created by Rafael on 1/16/2017.
 */
public class NetworkFunctionalTest {

    @Test
    public void test() throws IOException {

//        NetworkServer server = new NetworkServer(9999);
//        server.start();
//
//        NetworkClient client1 = new NetworkClient("localhost", 9999);
//        client1.addListener(event -> { handleEvent(event); });
//        client1.connect();
//        client1.send(data, responseBytes -> handle(responseBytes));
//        client1.send(data);
//
//        NetworkClient client2 = new NetworkClient("localhost", 9999);
//        client2.addListener(event -> { handleEvent(event); });
//        client2.connect();
//        client2.send(data, responseBytes -> handle(responseBytes));
//        client2.send(data);
//
//        server.broadcast(bytes);

        // compare received bytes
    }
}

